api/v1beta1/flinkcluster_types.go

```diff
@@ -154,6 +173,12 @@ type JobManagerSpec struct {
 	// Ports.
 	Ports JobManagerPorts `json:"ports,omitempty"`
 
+	// +patchMergeKey=containerPort
+	// +patchStrategy=merge
+
+	// Additional ports.
```

Comment:
Add a comment about the common use case of this field.

Reply:
Done.